src/gentropy/config.py

```diff
@@ -51,6 +51,15 @@ class GeneIndexConfig(StepConfig):
     _target_: str = "gentropy.gene_index.GeneIndexStep"
 
 
+@dataclass
+class BiosampleIndexConfig(StepConfig):
+    """Biosample index step configuration."""
+
+    target_path: str = MISSING
+    biosample_index_path: str = MISSING
+    _target_: str = "gentropy.biosample_index.BiosampleIndexStep"
```

Comment:
The configuration should have all the inputs that the step needs. 

- `cell_ontology_input_path`
- `uberon_input_path`
- `biosample_index_output_path`

All can/should be missing in this case, because there's no hardcoded parameters eg. [r2 threshold for LD index](https://github.com/opentargets/gentropy/blob/c3651b7e8ecad3631fce0d4e9fa79fea1b59d40a/src/gentropy/config.py#L173). Similarly, `target_path` needs to be dropped as this parameter is not part of the `BiosampleIndexStep` parameters.